pkg/ebpf/tracer.go

```diff
@@ -368,8 +370,33 @@ func (t *Tracer) GetActiveConnections(clientID string) (*network.Connections, er
 
 	conns := t.state.Connections(clientID, latestTime, latestConns, t.reverseDNS.GetDNSStats())
 	names := t.reverseDNS.Resolve(conns)
+	tm := t.getConnTelemetry(len(latestConns))
 
-	return &network.Connections{Conns: conns, DNS: names}, nil
+	return &network.Connections{Conns: conns, DNS: names, Telemetry: tm}, nil
+}
+
+func (t *Tracer) getConnTelemetry(mapSize int) *network.ConnectionsTelemetry {
+	kprobeStats := GetProbeTotals()
+	tm := &network.ConnectionsTelemetry{
+		MonotonicKprobesTriggered: kprobeStats.hits,
+		MonotonicKprobesMissed:    kprobeStats.miss,
+		ConnsBpfMapSize:           int64(mapSize),
+		MonotonicConnsClosed:      atomic.LoadInt64(&t.closedConns),
+	}
+
+	conntrackStats := t.conntracker.GetStats()
+	if rt, ok := conntrackStats["registers_total"]; ok {
```

Comment:
It'll be very easy for us to break this code path since these maps are untyped, can we add some `log.Warn()`s so it's clearer when this breaks

Reply:
If there is no data, the logs would result in false positives, due to this code:
https://github.com/DataDog/datadog-agent/blob/034883cebd50d941720a120dc1cd69f68175d81c/pkg/network/netlink/conntracker.go#L158-L162